plugins/package-managers/bundler/src/main/kotlin/Bundler.kt

```diff
@@ -359,7 +359,8 @@ class Bundler(
 
     private fun queryRubygems(name: String, version: String, retryCount: Int = 3): GemSpec? {
         // See http://guides.rubygems.org/rubygems-org-api-v2/.
-        val url = "https://rubygems.org/api/v2/rubygems/$name/versions/$version.yaml"
+        // See discussion in https://github.com/oss-review-toolkit/ort/issues/8695 on why we use the "platform=ruby".
```

Comment:
What do you think about adjusting the comment to summarize the rationale, and then link the discussion with a "see also $link"? ...Because, the whole discussion is quite lengthy to read through.

Reply:
I'd actually prefer to not link to our own issue discussions at all, but keep the issue self-contained. To me, the gist is to make the platform when querying metadarta consistent with the platform we resolve dependencies for at

https://github.com/oss-review-toolkit/ort/blob/503edeedc8de2bee866e2d308ac5473b69f1d42e/plugins/package-managers/bundler/src/main/resources/resolve_dependencies.rb#L31-L32